### PR TITLE
Revert "65 input form elements"

### PR DESCRIPTION
**Pull request by ghost**
Reverts deepumandal/vite-template#73

### CODE REVIEW

**Review by deepumandal — COMMENTED** [review]
why changes in ts and tsx and yml files ?

**Comment by RubyPdx** [comment]
> why changes in ts and tsx and yml files ?

Maine na kiya 😶 change kuch bhi

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -38,9 +35,6 @@ importers:
       '@radix-ui/react-icons':
         specifier: ^1.3.0
         version: 1.3.0(react@18.3.1)
-      '@radix-ui/react-label':
-        specifier: ^2.1.0
```
Comment (deepumandal) [inline pnpm-lock.yaml:42]: @RubyPdx changes

File: src/components/shared/Label/index.ts [hunk 2]
```diff
@@ -1 +0,0 @@
-export * from "./Label";
```
Comment (deepumandal) [inline src/components/shared/Label/index.ts:1]: ....

File: src/App.tsx [hunk 3]
```diff
@@ -338,34 +333,6 @@ const App = () => (
...
-
-  return (
-    <Flex asElement="section">
-      <Form {...form}>
-        <FormField
-          control={form.control}
-          name="username"
-          render={({ field }) => (
-            <FormItem>
-              <FormLabel>Username</FormLabel>
```
Comment (deepumandal) [inline src/App.tsx:352]: .... @RubyPdx

**Comment by deepumandal** [comment]
@RubyPdx Pr m tha ky

**Comment by RubyPdx** [comment]
> @RubyPdx Pr m tha ky

Maine bus readme ka Edit Kiya tha uske alava kuch nhi kiya🙂Hanito

**Comment by RubyPdx** [comment]
> @RubyPdx Pr m tha ky

Tu hi kiya hoga maine kisi file ke sath Chedkhani nhi ki

**Comment by deepumandal** [comment]
changes tere hai sare ...

**Comment by RubyPdx** [comment]
> changes tere hai sare ...

Pata nahi Meko 🙂 maine last time dekha tha markdown me kuch chize Tune thik se use nahi kiya to bus readme edit Kiya tha aur kuch nahi kasam se🥲 kuch ched khani nhi ki maine

**Comment by deepumandal** [comment]
okay 
pr close kr di h mene ye bhot old thi

**Comment by RubyPdx** [comment]
> okay pr close kr di h mene ye bhot old thi

Khamakha badnam kiya tune mujhe 🙂

**Pull request closed without merging**